packages/python/src/components/PyDoc.tsx

```diff
@@ -258,18 +266,18 @@ export interface PyDocExampleProps {
  * Create a PyDoc example, which is prepended by \>\>.
  */
 export function PyDocExample(props: PyDocExampleProps) {
-  const children = childrenArray(() => props.children);
+  const children = computed(() => childrenArray(() => props.children));
```

Comment:
I find it surprising, given the number of times that we're using this statement in this PR, not to find it anywhere in the existing Alloy code.

Makes me suspect that we either shouldn't be doing this, or that there's a better way to accomplish the same thing.

Reply:
So I then chose to add that.